website/content/docs/platform/k8s/csi/index.mdx

```diff
@@ -35,6 +35,8 @@ The following features are supported by the Vault CSI Provider:
 - Syncing secrets to Kubernetes secrets to be used as environment variables.
 - Installation via [Vault Helm](/vault/docs/platform/k8s/helm)
 
+@include kubernetes-supported-versions.mdx
```

Comment:
I think the filename needs to be quoted for some reason? These aren't rendering right now: https://vault-ecjile9xv-hashicorp.vercel.app/vault/docs/platform/k8s/csi#features

```suggestion
@include 'kubernetes-supported-versions.mdx'
```